structset_test.go

```diff
@@ -38,6 +38,27 @@ func TestStructset(t *testing.T) {
 			Name: "Luffy",
 		}
 		doc      = NewDocument(&user)
+		mutation = Mutation{
+			Cascade: true,
+			Mutates: map[string]Mutate{
+				"id":         Set("id", 1),
+				"name":       Set("name", "Luffy"),
+				"age":        Set("age", 0),
+				"created_at": Set("created_at", now()),
+				"updated_at": Set("updated_at", now()),
+			},
+		}
+	)
+
+	assert.Equal(t, mutation, Apply(doc, NewStructset(&user, false)))
+}
+
+func TestStructsetSkipZeroPrimaryKey(t *testing.T) {
```

Comment:
```suggestion
func TestStructset_skipZeroPrimaryKey(t *testing.T) {
```
a bit test naming of convention